apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -578,13 +584,97 @@ describe('BillingReceipt', () => {
             expect(objs).toHaveLength(0)
         })
 
+        test('resident can read his own billingReceipts', async () => {
+
+            const { residentClient, receipt } = await makeResidentClientWithOwnReceipt()
+
+            const objs = await BillingReceipt.getAll(residentClient, {}, { raw:true })
+            const obj = objs.data.objs[0]
+
+            expect(objs.data.objs).toHaveLength(1)
+            expect(obj.id).toEqual(receipt.id)
+            expect(obj.context).toBeNull()
+            expect(obj.property).toBeNull()
+            expect(obj.account).toBeNull()
+            expect(obj.importId).toBeNull()
+            expect(obj.importId).toBeNull()
+            expect(obj.raw).toBeNull()
+            expect(obj.category).not.toBeNull()
+            expect(obj.toPay).not.toBeNull()
+            expect(obj.toPayDetails).not.toBeNull()
+            expect(obj.services).not.toBeNull()
+            expect(obj.recipient).not.toBeNull()
+        })
+
+        test('resident can read his own billingReceipts if he has multiple residents and multiple billingAccounts', async () => {
```

Comment:
It's unclear what does this sentence means. How a resident can have multiple residents?